### DISCUSSION TITLE
How to disable progress bar in neural prophet

### DISCUSSION THREAD

**Discussion by SabarilalS** [ourownstory/neural_prophet#654] [Q&A - get help using NeuralProphet]
I would like to disable the progress bar that is showing while running neural prophet.
Even if I give progress='print' as the argument while calling fit method this progress bar is coming.
Is there any way to hide this in current version of neural prophet.?
![np](https://user-images.githubusercontent.com/107095162/172533933-42cf63fa-d150-4bee-9917-8b60edaf1cf5.JPG)

**Comment by wjunneng** [comment]
set_log_level("CRITICAL")
